core/client/fs/src/main/java/alluxio/client/file/RetryHandlingFileSystemMasterClient.java

```diff
@@ -143,12 +143,16 @@ public Void call() throws TException {
   @Override
   public synchronized void delete(final AlluxioURI path, final DeleteOptions options)
       throws IOException {
-    retryRPC(new RpcCallable<Void>() {
-      @Override
-      public Void call() throws TException {
-        mClient.remove(path.getPath(), options.isRecursive(), options.toThrift());
-        return null;
-      }
+    // retryRPC(new RpcCallable<Void>() {
```

Comment:
Please remove the commented-out code. We can always use the git history if we want to revert back to it.